bucket/prismlauncher-qt5.json

```diff
@@ -3,12 +3,27 @@
     "description": "An open source Minecraft launcher with the ability to manage multiple instances, accounts and mods. Focused on user freedom and free redistributability. Qt 5 build that still supports legacy Windows like 8.1 and 7.",
     "homepage": "https://prismlauncher.org/",
     "license": "GPL-3.0-only",
-    "notes": "To add Prism Launcher file association options for .ZIPs and .MRPACKS, run this: \"$dir\\install-associations.reg\"",
-    "url": "https://github.com/PrismLauncher/PrismLauncher/releases/download/6.3/PrismLauncher-Windows-MSVC-Legacy-6.3.zip",
-    "hash": "003b54560935c341ab825d41962323e56d649ea15a29f69d4ce64b93326c866c",
+    "notes": [
+        "To add Prism Launcher file association options for .ZIPs and .MRPACKS, run this: \"$dir\\install-associations.reg\"",
+        "This package is now using the portable version of Prism Launcher, and data should have been migrated automatically. If you are using a global install, you will need to copy your own data to the new Scoop persist directory"
+    ],
+    "url": "https://github.com/PrismLauncher/PrismLauncher/releases/download/6.3/PrismLauncher-Windows-MSVC-Legacy-Portable-6.3.zip",
+    "hash": "521804443e78a68cdc62f6f1645756f8d74a15ab1941c7ff48a7deb37d419ee3",
     "suggest": {
         "Microsoft Visual C++ Redistributables": "extras/vcredist2022"
     },
+    "pre_install": [
+        "$appdataPath = \"$HOME\\AppData\\Roaming\\PrismLauncher\"",
+        "if ( (Test-Path -Path $appdataPath\\*) -and (!(Test-Path -Path $persist_dir\\*)) -and (!($global)) ) {",
+        "    Write-Host \"Migrating data from $appdataPath to \"$persist_dir\" (this may take a while)\"",
```

Comment:
The `\"$persist_dir\"`'s quotes are not interpreted as quotes

![vmware_XE0dRuv3uo](https://github.com/Calinou/scoop-games/assets/65787561/43b054c9-a88a-45f6-bc75-42fbfe3fe3b1)

Also, it's probably best to make it a warning or coloured, just so that it's easier to see in the Scoop output.

Reply:
it wasn't meant to be in quotes, not sure why i did that lol. i am using `Write-Warning` now though